src/main/java/duke/io/Storage.java

```diff
@@ -0,0 +1,92 @@
+package duke.io;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import duke.exception.DukeException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.task.Todo;
+
+/**
+ * Encapsulates the methods used to read and store data onto disk.
+ */
+public class Storage {
+    private final String filePath;
+
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    /**
+     * Loads task list from file path.
+     */
+    public ArrayList<Task> load() throws DukeException {
```

Comment:
Should the javadoc for this method include @throws? I believe this would help other programmers to better understand this method.